python/pyspark/testing/utils.py

```diff
@@ -221,7 +221,130 @@ def check_error(
         )
 
 
-def assertDataFrameEqual(df: DataFrame, expected: DataFrame, check_row_order: bool = False):
+def assertSchemaEqual(
+    df_schema: StructType, expected_schema: StructType, ignore_nullable: bool = False
```

Comment:
I would name it like `this` vs `that` or `schema1` `schema2` or `actualSchema` vs `expectedSchema` or `actual` vs `expected`.